ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -101,11 +110,16 @@ KEXT_STATIC bool ShouldHandleFileOpEvent(
     VirtualizationRootHandle* root,
     int* pid);
 
+static void ResizePendingRenames(uint32_t newMaxPendingRenames);
+
 // State
 static kauth_listener_t s_vnodeListener = nullptr;
 static kauth_listener_t s_fileopListener = nullptr;
 
 static atomic_int s_numActiveKauthEvents;
+static SpinLock s_renameLock;
+static PendingRenameOperation* s_pendingRenames = nullptr;
+static uint32_t s_maxPendingRenames = 0;
```

Comment:
How about s_pendingRenamesSize versus s_maxPendingRenames?

Reply:
I'm not saying it's the best naming convention, but we've [used this naming pattern elsewhere](https://github.com/microsoft/VFSForGit/blob/8c61c1ad051c97545a2dd2b6b0e22fca9b3b7fa1/ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp#L26).

The other thing about "size" is that it's potentially ambiguous. Is it a number of array items? Size in bytes?

Reply:
Fair point, there is precedence.  I know what it means now.